ring/ring_benchmark_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func BenchmarkRing(b *testing.B) {
-	b.Run("GenRingContext", benchGenRingContext)
+	//b.Run("GenRingContext", benchGenRingContext)
```

Comment:
Is there a reason for removing this benchmark ?

Reply:
No, my bad